### PR TITLE
Review/Refactor of precedence table

### PR DESCRIPTION
**Pull request by ulyssessouza**
### Proposed changes
Review on environment variables precedence table

### CODE REVIEW

**Review by dockertopia — APPROVED** [review]
Thanks for the follow-up @ulyssessouza. Added some edits to better map the new column names and the description list, modified the note and added a stem sentence for the rule list. Please, if you find any difference in the values as expressed in this example fashion from your original table, create a PR or let me know.

**Comment by ulyssessouza** [comment]
LGTM

**Comment by dockertopia** [comment]
Closing and opening to trigger checks.

**Comment by thaJeztah** [comment]
@ulyssessouza could you do a rebase? I think this may be because some checks were added/removed in GitHub actions; a rebase hopefully should fix that)

File: compose/envvars-precedence.md [hunk 1]
```diff
@@ -5,38 +5,43 @@ keywords: compose, environment, env file
...
-| 16| -                          |  -                            |     `TAG=1.7`                       |     -        |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.4`           |
+|  # |  `.env` file      |  OS environment    |    Image    |  `compose.yaml`:`env_file` key  |  `compose.yaml`:`environment` key  |  `run --env`  |  Resolved as  |
+|:--:|:-----------------:|:------------------:|:-----------:|:-------------------------------:|:----------------------------------:|:-------------:|:-------------:|
+|  1 |  `TAG=1.3`        |  `TAG=1.4`         |   -         |   -                             |   -                                |   -           |   -           |
+|  2 |  `TAG=1.3`        |  `TAG=1.4`         |**`TAG=1.5`**|   -                             |   -                                |   -           |  `TAG=1.5`    |
+|  3 |  `TAG=1.3`        |**`TAG=1.4`**       |  `TAG=1.5`  |   -                             |   -                                |**`TAG`**      |  `TAG=1.4`    |
+|  4 |**`TAG=1.3`**      |   -                |  `TAG=1.5`  |**`TAG`**                        |   -                                |   -           |  `TAG=1.3`    |
```
Comment (thaJeztah) [inline compose/envvars-precedence.md:24]: wondering if these need spaces between the `|` and the content (`**...`)

Comment (ulyssessouza) [reply compose/envvars-precedence.md:23]: Not needed. It renders just fine -> https://deploy-preview-15425--docsdocker.netlify.app/compose/envvars-precedence/

**Comment by dockertopia** [comment]
Rebased https://github.com/docker/docker.github.io/pull/15511.

**Pull request closed without merging**